tests/testthat/test-utils.R

```diff
@@ -45,33 +45,217 @@ test_that("dims to col & row and back", {
   expect_equal(exp, got)
 
 })
+test_that("`wb_dims()` works/errors as expected with unnamed arguments", {
 
-test_that("wb_dims() works", {
+  # Acceptable inputs
+  expect_equal(wb_dims(), "A1")
+  expect_equal(wb_dims(NULL), "A1") # to help programming with `wb_dims()` maybe?
+  expect_equal(wb_dims(1L, 1L), "A1")
+  expect_equal(wb_dims(1:10, 1:26), "A1:Z10")
+  expect_equal(wb_dims(1:10, LETTERS), "A1:Z10")
+  expect_equal(
+    wb_dims(1:10, 1:12, from_row = 2),
+    wb_dims(rows = 1:10, cols = 1:12, from_row = 2)
+  )
+  expect_warning(wb_dims("1", 2))
+  expect_warning(wb_dims(from_row = "C"))
 
-  expect_equal(wb_dims(mtcars), "A1:K33")
-  expect_equal(wb_dims(mtcars, col_names = FALSE, row_names = TRUE), "A1:L32")
+  # Ambiguous / input not accepted.
+  # This now fails, as it used not to work. (Use `wb_dims()`, `NULL`, or )
+  expect_error(wb_dims(1), "Supplying a single unnamed argument.")
 
-  expect_equal(wb_dims(letters), "A1:A26")
+  # This used to return A1 as well.
+  expect_error(wb_dims(2), "Supplying a single unnamed argument is not handled")
+  expect_error(wb_dims(mtcars), "Supplying a single unnamed argument")
 
-  expect_equal(wb_dims(t(letters)), "A1:Z2")
 
-  expect_equal(wb_dims(1), "A1")
+  skip("lower priority, but giving non-consecutive rows, or cols should error in `wb_dims()`")
```

Comment:
Why should this be skipped?

Reply:
It still doesn't error / warn if users provide `wb_dims(c(1, 3, 4), 1:4)`

Reply:
done!